explainaboard/processors/kg_link_tail_prediction.py

```diff
@@ -111,6 +113,17 @@ def default_features(cls) -> feature.Features:
     def default_metrics(cls) -> list[str]:
         return ["Hits", "MeanReciprocalRank"]
 
+    def _get_metrics(
+        self, sys_info: SysOutputInfo
+    ) -> list[explainaboard.metric.Metric]:
+        config = explainaboard.metric.HitsConfig(
+            hits_k=5
```

Comment:
Two comments:

Shall we do the following instead?
`config = HitsConfig(...) if name == 'Hits' else MetricConfig()`

5 is already the default elsewhere, so we probably shouldn't set it here. If it is set here, then even if we change the default in `metric.py` the resulting value would not change, which could be confusing.

Reply:
sounds good!